packages/notification-store/README.md

```diff
@@ -1,20 +1,29 @@
-# Notification Store
+# ACPaaS UI JS Notification Store
 
 [![js-notification-store-status]][js-notification-store-package]
 
-The `aui-notification-store` provides a singleton store to save and easily access notifications throughout your app. You can load up a message map and trigger notifications by handle.
+The `@acpaas-ui/js-notification-store` package provides a singleton store to save and easily access notifications throughout your app. You can load up a message map and trigger notifications by handle.
 On top of that, you can set targets to hold your notifications (e.g. popups, statusbar, forms). Notifications are returned as `BehaviorSubjects`, allowing you to subscribe to changes.
 
 ## Installation
+---
 
-Install the package as a dependency:
+**NPM**
 ```
-npm install aui-notification-store --save
+npm i -S @acpaas-ui/js-notification-store
```

Comment:
I would prefer the extended style here: `npm install @acpaas-ui/js-notification-store --save`

Reply:
I went for this notation style to be consist with the other packages in this repository.
So if I update this one I would suggest to update the other README files as well.

Reply:
Ja, ik zou het dan eerder omdraaien a.u.b., omdat we dat zo in al onze andere repo's ook doen.

Reply:
Ok, I would do it the other way around then, since we're doing it like that in all our other repositories.